app/errors/index.js

```diff
@@ -1,3 +1,8 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
```

Comment:
Great custom error handler, love that you've split it into it's own file 👍 
This file will benefit from a PSQL error handler when you add the 400 test 🔍